matsim/simulation/prepare.py

```diff
@@ -133,26 +133,35 @@ def execute(context):
             "--factor", str(0.8)
         ])
 
-    # Route population
-    eqasim.run(context, "org.eqasim.core.scenario.routing.RunPopulationRouting", [
-        "--config-path", "%sconfig.xml" % context.config("output_prefix"),
-        "--output-path", "%spopulation.xml.gz" % context.config("output_prefix"),
-        "--threads", context.config("processes"),
-        "--config:plans.inputPlansFile", "prepared_population.xml.gz"
-    ])
-    assert os.path.exists("%s/%spopulation.xml.gz" % (context.path(), context.config("output_prefix")))
     
     # Optionally, perform mode choice
     if context.config("mode_choice"):
-        eqasim.run(context, "org.eqasim.ile_de_france.RunModeChoice", [
+        eqasim.run(context, "org.eqasim.ile_de_france.standalone_mode_choice.RunStandaloneModeChoice", [
             "--config-path", "%sconfig.xml" % context.config("output_prefix"),
-            "--output-plans-path", "%spopulation.xml.gz" % context.config("output_prefix"),
+            "--config:standaloneModeChoice.outputDirectory", "mode_choice",
+            "--config:standaloneModeChoice.removePersonsWithNoValidAlternatives", "true",
```

Comment:
Can you comment again how this can happen? Why won't people not just use walk?

Reply:
In the current version it has no effect, I had added this line quite some time ago for testing and left it as future proofing for when we would implement a constraint forbidding alternatives with very extensive delays (like arriving much after simulation end time). 
Should I remove this line for now ? 
For the record the case where such problems were observed was independent of synpp where a cut-out simulation contains a car-only trip chain for a person that should not have access to the car mode (but has car trips because it was generated during synpp and didn't change in the base simulation due to the randomness or the TourLengthFilter). This can result in some problems in the cut-out simulation if we want to extand the mode choice to outside-related tours.

Reply:
Ok, maybe remove it for now and we add it back in when it becomes necessary?

Reply:
Done